endpoints-common/src/main/scala/com/thenewmotion/ocpi/common/PaginatedRoute.scala

```diff
@@ -36,6 +35,8 @@ trait PaginatedRoute extends Directives with DateDeserializers {
 
   val paged = (offset & limit).as(Pager) & dateFrom & dateTo
 
+  def linkHeaderScheme: Option[String] = None
```

Comment:
if this is meant to handle http and https then we could make it more restrictive than `String`